third_party/openssl/Makefile

```diff
@@ -0,0 +1,35 @@
+TOP=$(abspath $(CURDIR)/../..)
+include $(TOP)/defs.mak
+
+URL=https://github.com/openssl/openssl
+
+BRANCH=OpenSSL_1_1_1-stable
```

Comment:
Another option is depending on a specific release tag (e.g., https://github.com/openssl/openssl/tree/OpenSSL_1_1_1k).

Reply:
Good idea! Done. :)